test/grisp_connect_test.hrl

```diff
@@ -0,0 +1,28 @@
+%--- Async asserts -------------------------------------------------------------
+
+% Spawn a process to wait for the result and check the result using assert.
+-define(asyncAssertEqual(Pattern, M, F, A),
+        spawn_monitor(?asyncAssertEqualFun(Pattern, M, F, A))).
+-define(asyncAssertEqualFun(Pattern, M, F, A),
+        fun() -> ?assertEqual(Pattern, erlang:apply(M, F, A)) end).
+
+% Use this macro to wait for the spawned process to end with normal status.
+-define(asyncWait(Pid, MRef),
+        ?assertEqual(normal, ?asyncWaitStatus(Pid, MRef))).
+-define(asyncWaitStatus(Pid, MRef),
```

Comment:
I would make the parameter opaque, the test do not have to know what is returned by asyncAssertEqual, so I would make the parameter `{Pid, Ref}`